src/notificationarea.h

```diff
@@ -54,13 +54,14 @@ class NotificationArea : public QScrollArea
      * \param unattendedMaxNum the max. number of unattended notifications to be saved
      * \param blackList the list of apps whose unattended notifications aren't saved
      */
-    void setSettings(const QString &placement, int width, int spacing, int unattendedMaxNum, const QStringList &blackList);
+    void setSettings(const QString &placement, int width, int spacing, int unattendedMaxNum, bool screen, const QStringList &blackList);
 
 private:
     NotificationLayout *m_layout;
 
     QString m_placement;
     int m_spacing;
+    bool m_screen;
```

Comment:
`m_screen` a boolean that shows whether we consider the screen with the mouse. Its name is misleading to the code reader (a name like "m_screen" is usually used as an integer that refers to as screen).

The same is true for the argument `bool screen` above it.

Please use names that show what the variables stand for.

Reply:
Should I use m_screenWithMouse then? that way it will definitely point out to "screenWithMouse"

Reply:
> Should I use m_screenWithMouse then?

Choose any name you see fit best. `m_screenWithMouse` looks good to me but this is *your* code.